sigs/cncf-sigs.md

```diff
@@ -0,0 +1,250 @@
+
+#### End User Input Gathering (Inbound Communication)
+
+    * Gather useful end user input and feedback regarding expectations, pain points, primary use cases etc.
+
+    * Compile this into easily consumable reports and/or presentations to assist projects with feature design, prioritization, UX etc.
+
+#### Community Enablement
+
+    * SIGs are open organizations with meetings, meeting agendas and notes, mailing lists, and other communications in the open
+
+    * The mailing list, SIG meeting calendar, and other communication documents of the SIG will be openly published and maintained
+
+#### As Trusted Expert Advisors to the TOC
+
+    * Perform technical due diligence on new and graduating projects, and advise TOC on findings.
+
+    * Be involved with, or periodically check in with projects in their area, and advise TOC on health, status and proposed actions (if any) as necessary or on request.
+
+#### SIG Charter:
+
+    * This is formally reviewed annually, and approved by the TOC.  The charter must clearly articulate:
+
+        * what is in and out of scope of the SIG,
+
+        * whether and how it overlaps and interfaces with other CNCF SIG’s or other relevant groups, and 
+
+        * how it operates and is governed, and specifically whether and how it deviates from standard SIG operating guidelines provided by the TOC.  Deviation from these guidelines is discouraged, unless there are good and well-documented reasons for such divergence, approved by the TOC. 
+
+See [Example Responsibilities of a CNCF SIG](https://docs.google.com/document/d/1L9dJl5aBFnN5KEf82J689FY0UtnUawnt9ooCq8SkO_w/edit?usp=sharing).
+
+## Operating Model
+
+Important: Each SIG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Executive Director, plus communication and performance of the SIG, with quarterly and annual reporting to Governing Board & TOC.
+
+As a starting point let’s be inspired by CNCF OSS Projects and by K8S SIGs.  That means minimal viable governance and community-based organisation.
+
+### SIG Formation, Leadership and Membership Composition
+
+1. SIGs are formed by the TOC.  Initial SIGs are listed below, and will be adapted over time as required.  If members of the community believe that additional SIGs are desired, they should propose these to the TOC, with clear justification, and ideally volunteers to lead the SIG. The TOC wishes to have the smallest viable number of SIGs, and for all of them to be highly effective (as opposed to a "SIG sprawl" with large numbers of relatively ineffective SIGS).
+
+2. SIG has three co-chairs, who are TOC Contributors, recognized as experts in that area, and for their ability to co-lead the SIG to produce the required unbiased outputs.
+
+3. SIG has one TOC liaison who is a voting member of the TOC acting as an additional non-executive chair on occasions when TOC input is deemed necessary by the TOC or the SIG chairs.
```

Comment:
@quinton-hoole-2 based on your comments on the TOC call today I wanted to elaborate on where I'm coming from with my concerns about this TOC liaison.  Based on my experience with the Serverless WG and CloudEvents, our interaction with the TOC is pretty minimal. Maybe we're odd in that respect, but I think we've only really talked with the TOC to update them on our status - which is pretty infrequent.  And I'm interpreting that as a good thing - we're just chugging along and trying to do our job.

I would hope that this is how most SIGs will operate. I'm assuming that the only time a SIG would need to get the TOC involved is when 1) some kind of formal decision needs to be made that requires a TOC vote, 2) a status update from the SIG, or 3) some controversial topic has some up at the SIG and they are unable to resolve it themselves - thus needing the TOC to weigh-in (this one is kind of related to 1).  But, in the end, I think all of these should get the entire TOC involved, not just one person's POV.

So, with that, I don't think the TOC will be continually pinged by the SIGs as you seemed to suggest during the TOC call today. If I'm wrong, then I question the effectiveness of that SIG  :-) 

Now, having said all of that... I do think having some kind of "TOC contact" or "champion" is useful to help provide guidance for what the TOC expects of them - but they're more there to provide "process guidance" rather than to "speak for the TOC".  Similar to the champions we have for WGs today.

Hopefully, that clarifies where my head is at on this.

Reply:
@quinton-hoole ^^^ not sure which ID to use :-)

Reply:
piggy back on Doug's comment here, does SIG co-chair has to be a TOC Contributor ? Would that leads to some limitation ?

Reply:
@hannibalhuang Yes, that point occurred to me too.  I didn't add that particular workding, but I think it's reasonable to ask SIG co-chairs to sign up as CNCF Contributors if they are not already.  The intention is not that anyone would be disqualified from being a SIG lead for not having been a CNCF contributor in the past.  On the contrary, one of the aims of this exercise is to increase the number of CNCF Contributors who are actually doing valuable work.  There exists a sentiment that many of the individuals currently listed as CNCF Contributors are in fact that in name only.

Reply:
@quinton-hoole-2 great :)